hazelcast-client/src/test/java/com/hazelcast/client/cache/nearcache/ClientNearCacheTestSupport.java

```diff
@@ -135,7 +113,7 @@ protected NearCacheTestContext createNearCacheTest(String cacheName, NearCacheCo
         NearCache<Data, String> nearCache =
                 nearCacheManager.getNearCache(cacheManager.getCacheNameWithPrefix(cacheName));
 
-        return new NearCacheTestContext(client, cacheManager, nearCacheManager, cache, nearCache);
+        return new NearCacheTestContext(client, null, cacheManager, null, nearCacheManager, cache, null, nearCache);
```

Comment:
May be an overloaded constructor (which only takes the non-null parameters) instead of using null for the parameters would look nicer.